OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -61,6 +61,10 @@ public class ArmamentInfo : PausableConditionalTraitInfo, Requires<AttackBaseInf
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		[GrantedConditionReference]
+		[Desc("Condition to grant while reloading.")]
+		public readonly string OnReloadingCondition = null;
```

Comment:
IMO this should be `ReloadingCondition` for consistency.  We don't do e.g. `OnAirborneCondition` or `OnCruisingCondition` in `Aircraft`.

Reply:
I was a bit unsure about this, because - purely from a wording perspective - some modders might misinterpret ReloadingCondition as "this condition is needed *for* reloading".
On the other hand, that's what the Desc is for, and modders should know from other traits that the property name would include "Requires" if that were its purpose.